src/_locales/ja/messages.json

```diff
@@ -260,28 +260,28 @@
     "wallets": "仮想通貨ウォレット",
     "untitled_wallet": "無題の財布",
     "your_message": "あなたのメッセージ...",
-    "plugin_file_service_display_name": "",
-    "plugin_file_service_file_key_copied": "",
-    "plugin_file_service_on_insert": "",
-    "plugin_file_service_uploading_preparing": "",
-    "plugin_file_service_uploading_estimating_time": "",
-    "plugin_file_service_uploading_in_minute_remaining": "",
-    "plugin_file_service_uploading_remaining": "",
-    "plugin_file_service_uploading_on_cancal": "",
-    "plugin_file_service_show_more": "",
-    "plugin_file_service_file_key": "",
-    "plugin_file_service_unencrypted": "",
-    "plugin_file_service_signing_failed": "",
-    "plugin_file_service_on_encrypt_it": "",
-    "plugin_file_service_on_show_more": "",
-    "plugin_file_service_legal_text": "",
-    "plugin_file_service_legal_terms_link": "",
-    "plugin_file_service_legal_policy_link": "",
-    "plugin_file_service_drop_indicator": "",
-    "plugin_file_service_drop_hint_1": "",
-    "plugin_file_service_drop_hint_2": "",
-    "plugin_file_service_drop_here": "",
-    "plugin_file_service_error_101": "",
-    "plugin_file_service_error_102": "",
-    "plugin_file_service_on_change_file": ""
+    "plugin_file_service_display_name": "プラグイン: ファイル転送",
+    "plugin_file_service_file_key_copied": "ファイルの鍵がコピーされました",
+    "plugin_file_service_on_insert": "ファイルを使用する",
+    "plugin_file_service_uploading_preparing": "準備しています",
+    "plugin_file_service_uploading_estimating_time": "予定準備時間...",
+    "plugin_file_service_uploading_in_minute_remaining": "{{seconds}}秒残っています",
+    "plugin_file_service_uploading_remaining": "{{minutes}}分{{seconds}}秒残っています",
+    "plugin_file_service_uploading_on_cancal": "アップロードしています...今このページを閉じないでください。",
+    "plugin_file_service_show_more": "他にも",
+    "plugin_file_service_file_key": "ファイルキー:",
+    "plugin_file_service_unencrypted": "このファイルは暗号化されていません",
+    "plugin_file_service_signing_failed": "サービスが使用できません。今しばらくお待ちください。",
+    "plugin_file_service_on_encrypt_it": "暗号化しよう",
+    "plugin_file_service_on_show_more": "サポートされていません",
+    "plugin_file_service_legal_text": "このプラグインを使用することで<terms>利用規約</terms>並びに<policy>個人情報保護方針（プライバシーポリシー）</policy>に同意したこととなります。",
+    "plugin_file_service_legal_terms_link": "https://legal.maskbook.com/arweave/file-service/plugin-terms.html",
+    "plugin_file_service_legal_policy_link": "https://legal.maskbook.com/arweave/file-service/privacy-policy-uploader.html",
+    "plugin_file_service_drop_indicator": "ファイルをドラッグ＆ドロップしてアップロード！",
+    "plugin_file_service_drop_hint_1": "ファイル容量の超過：{{limit}}.",
```

Comment:
the `：` is full-width symbol,
but `plugin_file_service_display_name` and `plugin_file_service_file_key` is half-width symbol 

why?

Reply:
@mizphses thanks.

Reply:
It's might be because of the IME's conversion... I should check before commit.

Reply:
@mizphses the case, use half-width symbol? or full-width symbol?
(i will fix it myself)